src/GitHub.App/ViewModels/ActorViewModel.cs

```diff
@@ -16,10 +17,30 @@ public ActorViewModel()
         public ActorViewModel(ActorModel model)
         {
             Login = model?.Login ?? "[unknown]";
-            Avatar = model?.AvatarUrl != null ?
-                new BitmapImage(new Uri(model.AvatarUrl)) :
-                AvatarProvider.CreateBitmapImage(DefaultAvatar);
-            AvatarUrl = model?.AvatarUrl ?? DefaultAvatar;
+
+            if (model?.AvatarUrl != null)
```

Comment:
Nit: I find the new `if (model?.AvatarUrl is string avatarUrl)` syntax more readable.

Reply:
Interesting, what do you find more readable about that? I use the new syntax a lot, but only when a cast is involved. `AvatarUrl` is already typed to string so there is no cast, and so it's just a comparison with null...

Reply:
Yeah, I dunno... The problem for me with "if model?.AvatarUrl is a string then call it avatarUrl and do stuff with it" is that it's _always_ a string. That pattern should be used for casting IMO, not just checking for null, and it surprises me when there isn't actually a cast.